python-sdk/nuscenes/eval/lidarseg/README.md

```diff
@@ -61,10 +62,10 @@ The folder structure of the results should be as follows:
 ```
 └── results_folder
     ├── lidarseg
-    │   └── v1.0-test <- Contains the .bin files; a .bin file 
+    │   └── test <- Contains the .bin files; a .bin file 
     │                    contains the labels of the points in a 
     │                    point cloud         
-    └── v1.0-test
+    └── test
         └── submission.json  <- contains certain information about 
                                 the submission
```

Comment:
The structure should be flexible for val, even though we don't use it on our eval server. Can you just omit the "test" folders?

Reply:
The intention of having these folders is so that users can store their predictions for `test`, `train` and `val` in the same root folder and also keep things consistent with the structure used in nuScenes-lidarseg. I've updated the readme to show this.

Reply:
Then let's write `{train,val,test}`